index.js

```diff
@@ -4,7 +4,35 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let command = process.argv[2];
+    let args = process.argv.length;
+    let sum = 0;
+    //checking for errors;
+    if(!command){
+        return "No operation provided...";
+    } 
+    else if(command !== "plus" && command !== "minus"){
+        return "Invalid operation: " + command;
+    }
+    else if(args < 4){
+        return "No numbers provided...";
+    }
+    //checking command
+    if(command === "plus"){
```

Comment:
The code in this if block and the one on line 27 are very similar with few differences. How can you refactor this code to be less repetitive??

Reply:
I can switch the logic up and instead of testing for both conditions, I can only test for one and base on whatever condition is true then the other will be false.